src/containers/app/App.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from '../../App';
 
 it('renders without crashing', () => {
```

Comment:
I see this is a test. However, You have not included steps to run it in the PR. You can consider that.

Reply:
I was setting up redux, @joma is supposed to ensure the tests are configured correctly.